platform/viewer/src/routes/WorkList/WorkList.jsx

```diff
@@ -364,17 +365,7 @@ function WorkList({ history, data: studies, dataSource }) {
           <span className="mr-3 text-lg text-common-light">
             FOR INVESTIGATIONAL USE ONLY
           </span>
```

Comment:
If the navbar is the same for two views, what do you think about a layout with the navbar included? so every new page would use the same layout and set the content of the navbar if needed, like longitudinal-workflow where the only thing that changed was the content in the middle of the navbar. This way we can reuse the investigational + navbar + preferencesdropdown code everywhere.

Reply:
had an offline conversation with @dannyrb and he said that we'll keep it this way for now and take care of it later.